OrbitGl/App.cpp

```diff
@@ -446,6 +464,12 @@ bool OrbitApp::Init() {
 //-----------------------------------------------------------------------------
 void OrbitApp::PostInit() {
   SetupIntrospection();
+
+  string_manager_ = std::make_shared<StringManager>();
+#ifndef NOGL
+  GCurrentTimeGraph->SetStringManager(string_manager_);
```

Comment:
I guess it has to be shared_ptr because GCurrentTimeGraph scope is global...

Reply:
Yes, I didn't really think about this deeply, but I think you are correct.